core/store/Cargo.toml

```diff
@@ -22,7 +22,7 @@ num_cpus = "1.11"
 rand = "0.7"
 strum = { version = "0.24", features = ["derive"] }
 fs2 = "0.4"
-tracing = "0.1.13"
+tracing = { version = "0.1.13", features = ["release_max_level_debug"] }
```

Comment:
I don't think it skews the results in any way, since timing is NOT something I want to record in this way. This trace is to have a record of _what_ happens exactly during execution. To get accurate timing info of the full system, the suggested feature should be turned off anyway. And to get timing info in controlled isolation, the traces would have to be replayed separately on the estimator.

That said, I think a new feature and some `cfg!` might be a better solution after all. I was just trying to avoid adding more crate features.